tools/telemetry2/thread-info.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
so now we're back to BSD-3-Clause? That doesn't seem consistent to me.

Reply:
I used the same license section that was used in mtrace-reader.py. What should I use, if not that?

Reply:
That's fine, the problem was the Apache stuff earlier.